project/Build.scala

```diff
@@ -598,7 +625,8 @@ object Build extends sbt.Build {
 
   lazy val delambdafySetting = {
     scalacOptions ++= (
-        if (scalaBinaryVersion.value == "2.10") Seq()
+        if (isGeneratingEclipse) Seq()
+        else if (scalaBinaryVersion.value == "2.10") Seq()
```

Comment:
I do not understand when you use `if (x || y) Seq() else Seq(...)` and when you use if (x) Seq() else if (y) Seq() else Seq(...)`